docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,476 @@
+    - API set
+      - efl_util_notification_window_level_error_cb
+      - efl_util_set_notification_window_level_error_cb
+      - efl_util_unset_notification_window_level_error_cb
+      - efl_util_window_screen_mode_error_cb
+      - efl_util_set_window_screen_mode_error_cb
+      - efl_util_unset_window_screen_mode_error_cb
+    - Enums
+      - EFL_UTIL_NOTIFICATION_LEVEL_1
+      - EFL_UTIL_NOTIFICATION_LEVEL_2
+      - EFL_UTIL_NOTIFICATION_LEVEL_3
+- libpepper-efl
+  - The wayland shell implementation has been replaced to use zxdg_shell_v6 interface.
+- Wayland TBM
+  - Added the set_buffer_serial request at the wl_tbm protocol.
+- Tizen Display HAL (TDM)
+  - Added the TDM HAL test cases.
+  - Added the tdm_hwc and tdm_hwc API for Vendor Driven HWC.
+- Tizen Buffer HAL (TBM)
+  - Added the TBM HAL test cases.
+- Tizen Porting Layer for EGL (TPL-EGL)
+  - libwayland-egl-tizen
+    - Added wl_egl_window_tizen_set_window_serial function.
+  - libtpl-egl
+    - Added tpl_surface_create_with_num_buffers function.
+    - Added tpl_display_get_with_backend_type function.
+- Vulkan
+  - Added support for VK_KHR_incremental_present.
+  - Added support for using OldSwapchain.
+
+### Fixes
+
+- Enlightenment (Display Server)
+  - Many code defects detected by the static analysis tool have been fixed.
+  - Fixed wrong input region of the window after the screen is rotated to the 90 degrees or 270 degrees.
+  - Fixed unwanted focus set to a window that is specified as 'focus_skip_set' when it is mapped.
+
+
+## Graphics Engine
+
+### New and changed features
+
+- DALi (3D UI Toolkit)
+  - Actor and Renderer
+    - Support for notification to inform Actor’s children order change has been added.
+    - Support for changing rendering behavior has been added.
+    - A mechanism to specify a callback on every frame has been added.
+    - A CULLED property has been added to Actor.
+  - Application
+    - IdleEnterer callback has been added.
+  - Input
+    - An API has been added to identify right and left mouse button click.
+    - Support for enabling the text prediction has been added.
+    - Support for keyboard repeat information has been added.
+  - Text
+    - Support for software styling has been added.
+  - Timer
+    - API set to pause and resume the timer has been added.
+  - Control and Visual
+    - WebView control has been added.
+    - Support for FlexLayout, AbsoluteLayout and GridLayout has been added.
+  - 3D Model
+    - Support for GL Transmission Format (glTF) has been added.
+  - Performance
+    - Changed not to render transparent texts.
+  - NUI (C# Interface)
+    - Added support for WebView.
+- Simple DirectMedia Layer (SDL)
+  - Upgrade to 2.0.8.
+  - Added SDL sub-module as SDL-ttf.
+  - Added new features.
+    - Auxiliary Hint.
+    - Focus Skip.
+
+### Fixes
+
+- DALi (3D UI Toolkit)
+  - Actor and Renderer
+    - RenderTask bug to waiting forever has been fixed.

```

Comment:
Can we rephrase as:
RenderTask bug (to waiting forever) has been fixed.

Reply:
will accept